### PR TITLE
LPS-148805 Prevent unpublished form to show on a page

### PR DESCRIPTION
**Pull request by leticia-maciel**
Related Issue: https://issues.liferay.com/browse/LPS-148805

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 1]
```diff
@@ -563,56 +563,47 @@ public boolean isAutosaveEnabled() throws PortalException {
...
 
 			return false;
 		}
 
-		if (formInstance != null) {
-			return true;
-		}
-
-		return false;
+		return true;
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:606]: @leticia-maciel can you please check if we can change to:

```
if (!isFormShared() && isSharedURL()) {
     return false;
}

return true;
```
Thanks!

Comment (leticia-maciel) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:606]: I'll test it now

Comment (leticia-maciel) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java]: Done!

**Comment by leticia-maciel** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by liferay-forms** [comment]
Hey @leticia-maciel, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**